hack/build.sh

```diff
@@ -81,7 +81,9 @@ run() {
 
 go_fmt() {
   echo "🧹 ${S}Format"
-  go fmt ./cmd/... ./pkg/...
+  gofmt -w $(find -name '*.go' | grep -v vendor)
```

Comment:
```suggestion
  find $(echo $source_dirs) -name "*.go" -print0 | xargs gofmt -w
```

Reply:
thanks!

Reply:
added -null option to xargs to suppress following warning 
```
xargs: WARNING: a NUL character occurred in the input.  It cannot be passed through in the argument list.  Did you mean to use the --null option?
```